src/woo_publications/publications/admin.py

```diff
@@ -312,6 +312,11 @@ class DocumentInlineAdmin(admin.StackedInline):
     extra = 0
 
 
+class PublicationIdentifierInlineAdmin(admin.StackedInline):
```

Comment:
let's go with a tabular layout here, it's more compact

```suggestion
class PublicationIdentifierInlineAdmin(admin.TabularInline):
```